README.md

```diff
@@ -0,0 +1,170 @@
+<p align="center">
+  <a href="https://delta.io/">
+    <img src="https://github.com/delta-io/delta-rs/blob/main/logo.png?raw=true" alt="delta-rs logo" height="250">
+  </a>
+</p>
+<p align="center">
+  A native Rust library for Delta Lake, with bindings into Python
+  <br>
+  <a href="https://delta-io.github.io/delta-rs/python/">Python docs</a>
+  ·
+  <a href="https://docs.rs/deltalake/latest/deltalake/">Rust docs</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/new?template=bug_report.md">Report a bug</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/new?template=feature_request.md">Request a feature</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/1128">Roadmap</a>
+  <br>
+  <br>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/l/deltalake.svg?style=flat-square&color=00ADD4&logo=apache">
+  </a>
+  <a target="_blank" href="https://github.com/delta-io/delta-rs" style="background:none">
+    <img src="https://img.shields.io/github/stars/delta-io/delta-rs?logo=github&color=F75101">
+  </a>
+  <a target="_blank" href="https://crates.io/crates/deltalake" style="background:none">
+    <img alt="Crate" src="https://img.shields.io/crates/v/deltalake.svg?style=flat-square&color=00ADD4&logo=rust" >
+  </a>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/v/deltalake.svg?style=flat-square&color=F75101&logo=pypi" >
+  </a>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/pyversions/deltalake.svg?style=flat-square&color=00ADD4&logo=python">
+  </a>
+  <a target="_blank" href="https://go.delta.io/slack">
+    <img alt="#delta-rs in the Delta Lake Slack workspace" src="https://img.shields.io/badge/slack-delta-blue.svg?logo=slack&style=flat-square&color=F75101">
+  </a>
```

Comment:
Could we add back a "Get involved section"?

Could list:

* Join the slack workspace (the button is nice, but it's kind of small)
* Report an issue
* Looking to contribute? See our list of good first issues (https://github.com/delta-io/delta-rs/contribute)